src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
And on those 9 large applications I can speak of whenever there is a mess of tags and presentational classes I can tell that those are hard to maintain, hard to extend, hard to do SEO and speed optimization on, slow on clients and demanding a lot of CPU and RAM as well as bandwith usually even having a ton of unnecessary http requests (because with the laziness usually comes including tons of plugins), being error prone and hard to integrate content management in an easy way and last but not least being vendor locked into whatever frontend framework used (in this case into Foundation, for us into Bootstrap). In my experience working on ecommerce and point of sale systems since 1995:

It is true that table-layouts / grid classes are simple to start with and use but hard to maintain and bad practice entirely. They are mostly, an excuse for not understanding DOM and CSS selectors. See https://www.ostraining.com/blog/coding/bootstrap-right-way/ for instance.